.spelling

```diff
@@ -220,20 +257,3 @@ workflow-controller-configmap
 WorkflowTemplate
 WorkflowTemplates
 yaml
-idempotence
```

Comment:
I alphabetized these.  WHERE THEY BELONG 😄

Reply:
Would like to alphabetize no matter case. thoughts?

Reply:
Automated this in #12521